modelmapper/stats.py

```diff
@@ -238,8 +238,8 @@ def inspect(self, field_name, item):
     def collect(self):
         return {'max_int': self.max_int }
 
-    def is_exclusive(self):
```

Comment:
A little documentation for why 6 would be helpful.
Also, was there a reason to leave out check for if `item is None`? Appears to me to be a type error waiting to happen.

Reply:
Good call. fixing that.